icon/iiss/icstate/timerCache.go

```diff
@@ -64,29 +64,30 @@ func (c *TimerCache) Clear() {
 }
 
 func (c *TimerCache) Reset() {
-	for _, timer := range c.timers {
-		value := c.dict.Get(timer.Height)
+	for key, timer := range c.timers {
+		value := c.dict.Get(key)
 
 		if value == nil {
 			timer.Clear()
 		} else {
 			timer.Set(ToTimer(value.Object()))
+			timer.dirty = false
```

Comment:
It's not a good idea to change the value of the internal variable.
If it clears dirty flag in Timer.Set(), then it's not required.